news/2021-06-12-new-featured-artist-a.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: A?"
+date: 2021-06-12 09:00:00 +0000
+---
+
+**A?** hops in as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/173/header.jpg)
+
+"Wouldn't it be cool to play an osu! map of my own music?" is a question surely every player has considered at some point. Few people actually go the extra mile to make it happen though, one example being J-core/breakcore producer **A?**, or [A__](https://osu.ppy.sh/users/12011880) on osu!.
```

Comment:
`one example being J-core/breakcore producer **A?**, otherwise known as *Aquestion* or [A__](https://osu.ppy.sh/users/12011880) on osu!.`

doesn't have to be in this sentence but i figured that it would be really nice to note that the artist also goes by the name "Aquestion" on many instances for convenience fwiw

![image](https://user-images.githubusercontent.com/36564236/121771034-25d2bc00-cb97-11eb-836e-2e0fdbfc5deb.png)

Reply:
(in fact i'm fairly certain that you can't search for this artist in osu! by searching for just "A?" -- you have to type in "Aquestion" instead)

![image](https://user-images.githubusercontent.com/36564236/121771123-9ed21380-cb97-11eb-89bf-5f4c33c75c13.png)

![image](https://user-images.githubusercontent.com/36564236/121771128-aa253f00-cb97-11eb-91ce-50b27eb6a054.png)

Reply:
applied this